ambuda/templates/proofing/create-project.html

```diff
@@ -13,8 +13,7 @@
 <div class="prose">
 <h1>Create new project</h1>
 
-<p>For now, please upload a list of images for your project. We hope to support
-PDF uploads soon.</p>
+<p>Please upload a PDF file of the original text.</p>
```

Comment:
or a list of images, if that happens to be convenient for some reason. (Don't necessarily have to assemble into a PDF)

Reply:
Done